ci/rust-version.sh

```diff
@@ -13,8 +13,18 @@
 #   $ source ci/rust-version.sh
 #
 
-stable_version=1.39.0
-nightly_version=2019-11-13
+if [ -n "$RUST_STABLE_VERSION" ]; then
```

Comment:
`[[ .. ]]` instead of `[ .. ]`please

Reply:
The extra quotes here are unnecessary too, so this could be written as:
```bash
if [[ -n $RUST_STABLE_VERSION ]]; then
```

sorry, I'm a bit of a bash snob

Reply:
Heh, I forgot about that. Actually, I personally like bashism too!